module3/homework/advanced_calculator/advancedCalculator.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <string>
+
+enum class ErrorCode {
+    OK,
+    BadCharacter,
+    BadFormat,
+    DivideBy0,
+    SqrtOfNegativeNumber,
+    ModuleOfNonIntegerValue
+};
+
+ErrorCode process(std::string input, double* out);
+
+//BadCharacters
```

Comment:
It is good idea to put there these functions declarations? (I mean checkCharackter, removes paces etc. ofc). You only use them in cpp file. What do you think about it?

Reply:
Now when I think about it you are absolutely right!